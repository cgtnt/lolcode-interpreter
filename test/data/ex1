COLOR, WTF?
  OMG "R"
    VISIBLE "RED FISH"
    GTFO
  OMG "Y"
    VISIBLE "YELLOW FISH"
  OMG "G"
  OMG "B"
    VISIBLE "FISH HAS A FLAVOR"
    GTFO
  OMGWTF
    VISIBLE "FISH IS TRANSPARENT"
OIC
