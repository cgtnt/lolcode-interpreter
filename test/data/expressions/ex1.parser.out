SUM OF 3 4